timeline/plugins.py

```diff
@@ -114,3 +114,28 @@ class SiteAppPlugin(SiteAppPluginPoint):
 
     #: Required permission for displaying the app
     app_permission = 'timeline.view_site_timeline'
+
+
+class AdminSiteAppPlugin(SiteAppPluginPoint):
+    """Projectroles plugin for registering the app"""
+
+    #: Name (slug-safe, used in URLs)
+    name = 'admin_site'
+
+    #: Title (used in templates)
+    title = 'Admin Site Events'
```

Comment:
"All Timeline Events" would probably be more descriptive here.

Reply:
- [x] Changed